executor/seqtest/seq_executor_test.go

```diff
@@ -774,6 +774,15 @@ func (s *seqTestSuite) TestUnparallelHashAggClose(c *C) {
 	req := rs.NewChunk()
 	err = rs.Next(ctx, req)
 	c.Assert(err.Error(), Equals, "HashAggExec.unparallelExec error")
+
+	tk.MustExec("set @@tidb_hashagg_partial_concurrency=1")
+	tk.MustExec("set @@tidb_hashagg_final_concurrency=1")
+	rss, err = tk.Se.Execute(ctx, "select sum(distinct a) from (select cast(t.a as signed) as a, b from t) t group by b;")
```

Comment:
The aggregate functions are not all firstrow.

Reply:
I have replace it with `select distinct a from t limit 1`.